lang/py/test/av_bench.py

```diff
@@ -18,36 +18,38 @@
 
 import sys
 import time
-from random import sample, choice, randint
+from random import choice, randint, sample
 from string import lowercase
 
 import avro.datafile
-import avro.schema
 import avro.io
-
+import avro.schema
```

Comment:
```suggestion
from avro import datafile, io, schema
```

Reply:
We have to be careful, because `io` is also a python core module, so we may want to keep using `avro.io` explicitly.

Reply:
(I'd never seen "suggested change" in action before, cool.)